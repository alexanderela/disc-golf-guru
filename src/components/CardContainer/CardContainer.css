.CardContainer {
	border: 3px solid #febb00;
	width: 50%;
	margin: 1rem auto;
	height: 20rem;
	background-color: #FFF;
	border-radius: 10px;
}