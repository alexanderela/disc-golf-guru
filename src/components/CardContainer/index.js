import React from 'react';
import Card from '../../containers/Card';

const CardContainer = () => {
	return(
		<div>
		</div>
	)
}

export default CardContainer