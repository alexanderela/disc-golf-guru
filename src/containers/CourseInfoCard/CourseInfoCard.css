.CourseInfoCard {
	border: 3px solid #febb00;
  color: #524949;
	width: 20rem;
  text-align: center;
	margin: 1.25rem auto;
	height: 24rem;
	background-color: #FFF;
	border-radius: 20px;
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  align-items: center;
  padding: 0 0 1rem 0;
  box-shadow: 0px 0px 1px 2px #Feeb00;
}

.header-container {
	display: flex;
	align-items: center;
	justify-content: space-between;
	width: 85%;
	margin: 1.25rem 0 0;
}

.favorite-btn {
	border-radius: 50%;
	width: 3rem;
	height: 3rem;
	display: flex;
	align-items: center;
	justify-content: center;
	background-color: #febb00;
	cursor: pointer;
	border: none;
	justify-self: right;
	margin: 0 0.25rem 0 0;
  outline-width: 0;
}

.favorite-btn:hover {
	background-color: #2c9a2c;
}

.favorite-btn:hover > .fa-heart {
	background-color: #2c9a2c;	
	color: #febb00;
}

.fa-heart {
	color: #2c9a2c;
	font-size: 1.5rem;
}

.course-name {
  font-size: 1.25rem;
  color: #2c9a2c;
  font-weight: 600;
  overflow-wrap: break-word;
	width: 77%;
}

.course-address {
  font-size: 1.1rem;
  line-height: 0.75rem;
	margin: 1.25rem 0 0;
}

.address-fields {
	margin: 1rem 0;
}

.course-info-header {
	font-weight: 700;
}

.course-info {
	margin: 0.8rem 0;
}