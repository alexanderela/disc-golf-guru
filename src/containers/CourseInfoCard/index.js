import React, { Component } from 'react';
import './CourseInfoCard.css';
import { connect } from 'react-redux';
import { toggleFavorite } from '../../actions/courseActions';

class CourseInfoCard extends Component {
	constructor(props) {
		super(props);
		this.state = {

		}
	}

	handleFavorite = async (course) => {
		this.props.toggleFavorite(course.id);
	}

	render() {
		const { course, displayWeather } = this.props;
		const {
			id, 
			name, 
			address, 
			city, state, 
			zip, 
			holes,
			rating,
			isPrivate,
			isPayToPlay
		} = course;

		return(
			<div className='CourseInfoCard'>
				<div className='header-container'>
					<h2 className='course-name'>{name}</h2>
					<button 
						className='favorite-btn'
						onClick={() => this.handleFavorite(course)}>
						<i className="fas fa-heart"></i>
					</button>
				</div>
				<div className='course-address'>
					<p className='address-fields'>{address}</p>
					<p className='address-fields'>{`${city}, ${state} ${zip}`}</p>
				</div>

				<p className='course-info'>
					<span className='course-info-header'>
						Number of holes:
					</span> {holes}
				</p>

				<p className='course-info'>
					<span className='course-info-header'>
						Rating: 
					</span> {rating}
				</p>

				<p className='course-info'>
					<span className='course-info-header'>
						Private: 
					</span> {isPrivate}
					</p>

				<p className='course-info'>
					<span className='course-info-header'>
						Pay to play: 
					</span> {isPayToPlay}
				</p>
			</div>
		)
	}
};

export const mapStateToProps = ({ golfCourses }) => ({ golfCourses });

export const mapDispatchToProps = (dispatch) => ({
	toggleFavorite: (courseId) => dispatch(toggleFavorite(courseId))
});


export default connect(mapStateToProps, mapDispatchToProps)(CourseInfoCard);