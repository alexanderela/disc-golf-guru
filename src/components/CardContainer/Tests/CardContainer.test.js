import React from 'react';
import ReactDOM from 'react-dom';
import CardContainer from '../';
import { shallow } from 'enzyme';
import mockCourses from '../../../mockData/mockCourses.js';
import CourseInfoCard from '../../../containers/CourseInfoCard';

describe('CardContainer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CardContainer favorites={mockCourses} />);
  });
  it('should render like snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render each card', () => {
    expect(wrapper.find(CourseInfoCard).length).toEqual(5);
  });
});
