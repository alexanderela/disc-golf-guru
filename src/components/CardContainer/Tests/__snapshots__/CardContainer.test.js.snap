// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContainer should render like snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CardContainer
    favorites={
      Array [
        Object {
          "city": "Rochester",
          "country": "United States",
          "course_id": "4712",
          "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=4712",
          "dgcr_url": "http://www.dgcoursereview.com/course.php?id=4712",
          "holes": "18",
          "latitude": "43.119428",
          "longitude": "-77.636679",
          "name": "Genesee Valley Park",
          "paytoplay": "0",
          "private": "0",
          "rating": "3.29",
          "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.5.png",
          "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.5.png",
          "reviews": "7",
          "state": "NY",
          "street_addr": "99 Elmwood Ave.",
          "tee_1_clr": "FF0000",
          "tee_2_clr": "",
          "tee_3_clr": "",
          "tee_4_clr": "",
          "zipcode": "14620",
        },
        Object {
          "city": "Henrietta",
          "country": "United States",
          "course_id": "7628",
          "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=7628",
          "dgcr_url": "http://www.dgcoursereview.com/course.php?id=7628",
          "holes": "9",
          "latitude": "43.045582",
          "longitude": "-77.617742",
          "name": "Lookup Park",
          "paytoplay": "0",
          "private": "0",
          "rating": "3.13",
          "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.png",
          "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.png",
          "reviews": "4",
          "state": "NY",
          "street_addr": "3850 East Henrietta Rd.",
          "tee_1_clr": "FF0000",
          "tee_2_clr": "",
          "tee_3_clr": "",
          "tee_4_clr": "",
          "zipcode": "14467",
        },
        Object {
          "city": "Henrietta",
          "country": "United States",
          "course_id": "4937",
          "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=4937",
          "dgcr_url": "http://www.dgcoursereview.com/course.php?id=4937",
          "holes": "18",
          "latitude": "43.103033",
          "longitude": "-77.605526",
          "name": "Monroe Community College DGC",
          "paytoplay": "0",
          "private": "0",
          "rating": "2.94",
          "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.png",
          "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.png",
          "reviews": "16",
          "state": "NY",
          "street_addr": "",
          "tee_1_clr": "FFFF00",
          "tee_2_clr": "000000",
          "tee_3_clr": "",
          "tee_4_clr": "",
          "zipcode": "14623",
        },
        Object {
          "city": "Rochester",
          "country": "United States",
          "course_id": "6795",
          "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=6795",
          "dgcr_url": "http://www.dgcoursereview.com/course.php?id=6795",
          "holes": "18",
          "latitude": "43.153154",
          "longitude": "-77.527428",
          "name": "Ellison Park",
          "paytoplay": "0",
          "private": "0",
          "rating": "3.54",
          "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.5.png",
          "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.5.png",
          "reviews": "12",
          "state": "NY",
          "street_addr": "",
          "tee_1_clr": "",
          "tee_2_clr": "",
          "tee_3_clr": "0000CC",
          "tee_4_clr": "",
          "zipcode": "14610",
        },
        Object {
          "city": "Webster",
          "country": "United States",
          "course_id": "6443",
          "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=6443",
          "dgcr_url": "http://www.dgcoursereview.com/course.php?id=6443",
          "holes": "9",
          "latitude": "43.216217",
          "longitude": "-77.439955",
          "name": "North Ponds DGC",
          "paytoplay": "0",
          "private": "0",
          "rating": "1.75",
          "rating_img": "http://www.dgcoursereview.com/images/rating/discs_2.png",
          "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_2.png",
          "reviews": "2",
          "state": "NY",
          "street_addr": "750 Holt Rd.",
          "tee_1_clr": "",
          "tee_2_clr": "FF9900",
          "tee_3_clr": "",
          "tee_4_clr": "",
          "zipcode": "14580",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Connect(CourseInfoCard)
          course={
            Object {
              "city": "Rochester",
              "country": "United States",
              "course_id": "4712",
              "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=4712",
              "dgcr_url": "http://www.dgcoursereview.com/course.php?id=4712",
              "holes": "18",
              "latitude": "43.119428",
              "longitude": "-77.636679",
              "name": "Genesee Valley Park",
              "paytoplay": "0",
              "private": "0",
              "rating": "3.29",
              "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.5.png",
              "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.5.png",
              "reviews": "7",
              "state": "NY",
              "street_addr": "99 Elmwood Ave.",
              "tee_1_clr": "FF0000",
              "tee_2_clr": "",
              "tee_3_clr": "",
              "tee_4_clr": "",
              "zipcode": "14620",
            }
          }
        />,
        <Connect(CourseInfoCard)
          course={
            Object {
              "city": "Henrietta",
              "country": "United States",
              "course_id": "7628",
              "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=7628",
              "dgcr_url": "http://www.dgcoursereview.com/course.php?id=7628",
              "holes": "9",
              "latitude": "43.045582",
              "longitude": "-77.617742",
              "name": "Lookup Park",
              "paytoplay": "0",
              "private": "0",
              "rating": "3.13",
              "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.png",
              "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.png",
              "reviews": "4",
              "state": "NY",
              "street_addr": "3850 East Henrietta Rd.",
              "tee_1_clr": "FF0000",
              "tee_2_clr": "",
              "tee_3_clr": "",
              "tee_4_clr": "",
              "zipcode": "14467",
            }
          }
        />,
        <Connect(CourseInfoCard)
          course={
            Object {
              "city": "Henrietta",
              "country": "United States",
              "course_id": "4937",
              "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=4937",
              "dgcr_url": "http://www.dgcoursereview.com/course.php?id=4937",
              "holes": "18",
              "latitude": "43.103033",
              "longitude": "-77.605526",
              "name": "Monroe Community College DGC",
              "paytoplay": "0",
              "private": "0",
              "rating": "2.94",
              "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.png",
              "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.png",
              "reviews": "16",
              "state": "NY",
              "street_addr": "",
              "tee_1_clr": "FFFF00",
              "tee_2_clr": "000000",
              "tee_3_clr": "",
              "tee_4_clr": "",
              "zipcode": "14623",
            }
          }
        />,
        <Connect(CourseInfoCard)
          course={
            Object {
              "city": "Rochester",
              "country": "United States",
              "course_id": "6795",
              "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=6795",
              "dgcr_url": "http://www.dgcoursereview.com/course.php?id=6795",
              "holes": "18",
              "latitude": "43.153154",
              "longitude": "-77.527428",
              "name": "Ellison Park",
              "paytoplay": "0",
              "private": "0",
              "rating": "3.54",
              "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.5.png",
              "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.5.png",
              "reviews": "12",
              "state": "NY",
              "street_addr": "",
              "tee_1_clr": "",
              "tee_2_clr": "",
              "tee_3_clr": "0000CC",
              "tee_4_clr": "",
              "zipcode": "14610",
            }
          }
        />,
        <Connect(CourseInfoCard)
          course={
            Object {
              "city": "Webster",
              "country": "United States",
              "course_id": "6443",
              "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=6443",
              "dgcr_url": "http://www.dgcoursereview.com/course.php?id=6443",
              "holes": "9",
              "latitude": "43.216217",
              "longitude": "-77.439955",
              "name": "North Ponds DGC",
              "paytoplay": "0",
              "private": "0",
              "rating": "1.75",
              "rating_img": "http://www.dgcoursereview.com/images/rating/discs_2.png",
              "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_2.png",
              "reviews": "2",
              "state": "NY",
              "street_addr": "750 Holt Rd.",
              "tee_1_clr": "",
              "tee_2_clr": "FF9900",
              "tee_3_clr": "",
              "tee_4_clr": "",
              "zipcode": "14580",
            }
          }
        />,
      ],
      "className": "CardContainer",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "course": Object {
            "city": "Rochester",
            "country": "United States",
            "course_id": "4712",
            "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=4712",
            "dgcr_url": "http://www.dgcoursereview.com/course.php?id=4712",
            "holes": "18",
            "latitude": "43.119428",
            "longitude": "-77.636679",
            "name": "Genesee Valley Park",
            "paytoplay": "0",
            "private": "0",
            "rating": "3.29",
            "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.5.png",
            "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.5.png",
            "reviews": "7",
            "state": "NY",
            "street_addr": "99 Elmwood Ave.",
            "tee_1_clr": "FF0000",
            "tee_2_clr": "",
            "tee_3_clr": "",
            "tee_4_clr": "",
            "zipcode": "14620",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "course": Object {
            "city": "Henrietta",
            "country": "United States",
            "course_id": "7628",
            "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=7628",
            "dgcr_url": "http://www.dgcoursereview.com/course.php?id=7628",
            "holes": "9",
            "latitude": "43.045582",
            "longitude": "-77.617742",
            "name": "Lookup Park",
            "paytoplay": "0",
            "private": "0",
            "rating": "3.13",
            "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.png",
            "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.png",
            "reviews": "4",
            "state": "NY",
            "street_addr": "3850 East Henrietta Rd.",
            "tee_1_clr": "FF0000",
            "tee_2_clr": "",
            "tee_3_clr": "",
            "tee_4_clr": "",
            "zipcode": "14467",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "course": Object {
            "city": "Henrietta",
            "country": "United States",
            "course_id": "4937",
            "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=4937",
            "dgcr_url": "http://www.dgcoursereview.com/course.php?id=4937",
            "holes": "18",
            "latitude": "43.103033",
            "longitude": "-77.605526",
            "name": "Monroe Community College DGC",
            "paytoplay": "0",
            "private": "0",
            "rating": "2.94",
            "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.png",
            "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.png",
            "reviews": "16",
            "state": "NY",
            "street_addr": "",
            "tee_1_clr": "FFFF00",
            "tee_2_clr": "000000",
            "tee_3_clr": "",
            "tee_4_clr": "",
            "zipcode": "14623",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "course": Object {
            "city": "Rochester",
            "country": "United States",
            "course_id": "6795",
            "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=6795",
            "dgcr_url": "http://www.dgcoursereview.com/course.php?id=6795",
            "holes": "18",
            "latitude": "43.153154",
            "longitude": "-77.527428",
            "name": "Ellison Park",
            "paytoplay": "0",
            "private": "0",
            "rating": "3.54",
            "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.5.png",
            "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.5.png",
            "reviews": "12",
            "state": "NY",
            "street_addr": "",
            "tee_1_clr": "",
            "tee_2_clr": "",
            "tee_3_clr": "0000CC",
            "tee_4_clr": "",
            "zipcode": "14610",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "course": Object {
            "city": "Webster",
            "country": "United States",
            "course_id": "6443",
            "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=6443",
            "dgcr_url": "http://www.dgcoursereview.com/course.php?id=6443",
            "holes": "9",
            "latitude": "43.216217",
            "longitude": "-77.439955",
            "name": "North Ponds DGC",
            "paytoplay": "0",
            "private": "0",
            "rating": "1.75",
            "rating_img": "http://www.dgcoursereview.com/images/rating/discs_2.png",
            "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_2.png",
            "reviews": "2",
            "state": "NY",
            "street_addr": "750 Holt Rd.",
            "tee_1_clr": "",
            "tee_2_clr": "FF9900",
            "tee_3_clr": "",
            "tee_4_clr": "",
            "zipcode": "14580",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Connect(CourseInfoCard)
            course={
              Object {
                "city": "Rochester",
                "country": "United States",
                "course_id": "4712",
                "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=4712",
                "dgcr_url": "http://www.dgcoursereview.com/course.php?id=4712",
                "holes": "18",
                "latitude": "43.119428",
                "longitude": "-77.636679",
                "name": "Genesee Valley Park",
                "paytoplay": "0",
                "private": "0",
                "rating": "3.29",
                "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.5.png",
                "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.5.png",
                "reviews": "7",
                "state": "NY",
                "street_addr": "99 Elmwood Ave.",
                "tee_1_clr": "FF0000",
                "tee_2_clr": "",
                "tee_3_clr": "",
                "tee_4_clr": "",
                "zipcode": "14620",
              }
            }
          />,
          <Connect(CourseInfoCard)
            course={
              Object {
                "city": "Henrietta",
                "country": "United States",
                "course_id": "7628",
                "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=7628",
                "dgcr_url": "http://www.dgcoursereview.com/course.php?id=7628",
                "holes": "9",
                "latitude": "43.045582",
                "longitude": "-77.617742",
                "name": "Lookup Park",
                "paytoplay": "0",
                "private": "0",
                "rating": "3.13",
                "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.png",
                "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.png",
                "reviews": "4",
                "state": "NY",
                "street_addr": "3850 East Henrietta Rd.",
                "tee_1_clr": "FF0000",
                "tee_2_clr": "",
                "tee_3_clr": "",
                "tee_4_clr": "",
                "zipcode": "14467",
              }
            }
          />,
          <Connect(CourseInfoCard)
            course={
              Object {
                "city": "Henrietta",
                "country": "United States",
                "course_id": "4937",
                "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=4937",
                "dgcr_url": "http://www.dgcoursereview.com/course.php?id=4937",
                "holes": "18",
                "latitude": "43.103033",
                "longitude": "-77.605526",
                "name": "Monroe Community College DGC",
                "paytoplay": "0",
                "private": "0",
                "rating": "2.94",
                "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.png",
                "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.png",
                "reviews": "16",
                "state": "NY",
                "street_addr": "",
                "tee_1_clr": "FFFF00",
                "tee_2_clr": "000000",
                "tee_3_clr": "",
                "tee_4_clr": "",
                "zipcode": "14623",
              }
            }
          />,
          <Connect(CourseInfoCard)
            course={
              Object {
                "city": "Rochester",
                "country": "United States",
                "course_id": "6795",
                "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=6795",
                "dgcr_url": "http://www.dgcoursereview.com/course.php?id=6795",
                "holes": "18",
                "latitude": "43.153154",
                "longitude": "-77.527428",
                "name": "Ellison Park",
                "paytoplay": "0",
                "private": "0",
                "rating": "3.54",
                "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.5.png",
                "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.5.png",
                "reviews": "12",
                "state": "NY",
                "street_addr": "",
                "tee_1_clr": "",
                "tee_2_clr": "",
                "tee_3_clr": "0000CC",
                "tee_4_clr": "",
                "zipcode": "14610",
              }
            }
          />,
          <Connect(CourseInfoCard)
            course={
              Object {
                "city": "Webster",
                "country": "United States",
                "course_id": "6443",
                "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=6443",
                "dgcr_url": "http://www.dgcoursereview.com/course.php?id=6443",
                "holes": "9",
                "latitude": "43.216217",
                "longitude": "-77.439955",
                "name": "North Ponds DGC",
                "paytoplay": "0",
                "private": "0",
                "rating": "1.75",
                "rating_img": "http://www.dgcoursereview.com/images/rating/discs_2.png",
                "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_2.png",
                "reviews": "2",
                "state": "NY",
                "street_addr": "750 Holt Rd.",
                "tee_1_clr": "",
                "tee_2_clr": "FF9900",
                "tee_3_clr": "",
                "tee_4_clr": "",
                "zipcode": "14580",
              }
            }
          />,
        ],
        "className": "CardContainer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "course": Object {
              "city": "Rochester",
              "country": "United States",
              "course_id": "4712",
              "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=4712",
              "dgcr_url": "http://www.dgcoursereview.com/course.php?id=4712",
              "holes": "18",
              "latitude": "43.119428",
              "longitude": "-77.636679",
              "name": "Genesee Valley Park",
              "paytoplay": "0",
              "private": "0",
              "rating": "3.29",
              "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.5.png",
              "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.5.png",
              "reviews": "7",
              "state": "NY",
              "street_addr": "99 Elmwood Ave.",
              "tee_1_clr": "FF0000",
              "tee_2_clr": "",
              "tee_3_clr": "",
              "tee_4_clr": "",
              "zipcode": "14620",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "course": Object {
              "city": "Henrietta",
              "country": "United States",
              "course_id": "7628",
              "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=7628",
              "dgcr_url": "http://www.dgcoursereview.com/course.php?id=7628",
              "holes": "9",
              "latitude": "43.045582",
              "longitude": "-77.617742",
              "name": "Lookup Park",
              "paytoplay": "0",
              "private": "0",
              "rating": "3.13",
              "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.png",
              "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.png",
              "reviews": "4",
              "state": "NY",
              "street_addr": "3850 East Henrietta Rd.",
              "tee_1_clr": "FF0000",
              "tee_2_clr": "",
              "tee_3_clr": "",
              "tee_4_clr": "",
              "zipcode": "14467",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "course": Object {
              "city": "Henrietta",
              "country": "United States",
              "course_id": "4937",
              "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=4937",
              "dgcr_url": "http://www.dgcoursereview.com/course.php?id=4937",
              "holes": "18",
              "latitude": "43.103033",
              "longitude": "-77.605526",
              "name": "Monroe Community College DGC",
              "paytoplay": "0",
              "private": "0",
              "rating": "2.94",
              "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.png",
              "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.png",
              "reviews": "16",
              "state": "NY",
              "street_addr": "",
              "tee_1_clr": "FFFF00",
              "tee_2_clr": "000000",
              "tee_3_clr": "",
              "tee_4_clr": "",
              "zipcode": "14623",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "course": Object {
              "city": "Rochester",
              "country": "United States",
              "course_id": "6795",
              "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=6795",
              "dgcr_url": "http://www.dgcoursereview.com/course.php?id=6795",
              "holes": "18",
              "latitude": "43.153154",
              "longitude": "-77.527428",
              "name": "Ellison Park",
              "paytoplay": "0",
              "private": "0",
              "rating": "3.54",
              "rating_img": "http://www.dgcoursereview.com/images/rating/discs_3.5.png",
              "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_3.5.png",
              "reviews": "12",
              "state": "NY",
              "street_addr": "",
              "tee_1_clr": "",
              "tee_2_clr": "",
              "tee_3_clr": "0000CC",
              "tee_4_clr": "",
              "zipcode": "14610",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "course": Object {
              "city": "Webster",
              "country": "United States",
              "course_id": "6443",
              "dgcr_mobile_url": "http://www.dgcoursereview.com/mobile/course.php?id=6443",
              "dgcr_url": "http://www.dgcoursereview.com/course.php?id=6443",
              "holes": "9",
              "latitude": "43.216217",
              "longitude": "-77.439955",
              "name": "North Ponds DGC",
              "paytoplay": "0",
              "private": "0",
              "rating": "1.75",
              "rating_img": "http://www.dgcoursereview.com/images/rating/discs_2.png",
              "rating_img_small": "http://www.dgcoursereview.com/images/rating/discs_sm_2.png",
              "reviews": "2",
              "state": "NY",
              "street_addr": "750 Holt Rd.",
              "tee_1_clr": "",
              "tee_2_clr": "FF9900",
              "tee_3_clr": "",
              "tee_4_clr": "",
              "zipcode": "14580",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
